appservices_local_dev.sh

```diff
@@ -31,23 +31,19 @@ if [[ "${ACTION}" == "enable" ]]; then
     echo "Please specify the application services directory to use!"
     exit 1
   fi
-  msg "Using local Application Services (${APP_SERVICES_DIR})"
-  if [ ! -h "${FRAMEWORK_LOCATION}" ]; then
-    msg "Replacing Carthage package by symbolic link..."
-    rm -rf "${FRAMEWORK_LOCATION}"
-    ln -s "${APP_SERVICES_DIR}/Carthage/Build/iOS/MozillaAppServices.framework" "${PWD}/Carthage/Build/iOS"
-  fi
   msg "Building Application Services."
   pushd "${APP_SERVICES_DIR}"
   ./build-carthage.sh --no-archive
   popd
   echo ""
+  msg "Copying local Application Services (${APP_SERVICES_DIR}) into Carthage build directory"
+  rsync -ad "${APP_SERVICES_DIR}/Carthage/Build/iOS/MozillaAppServices.framework/" "${FRAMEWORK_LOCATION}/"
```

Comment:
Since we've moved to using `carthage copy-framework` on the appservices framework, this setup with the symlink no longer works - Carthage tries to be clever and dereference the symlink and gets upset that it's pointing outside of the build root. Having a little script here that can physically rebuilt and rsync to the right place still seems useful though.

Reply:
I moved this into a separate PR at https://github.com/mozilla-mobile/firefox-ios/pull/8081